### ISSUE TITLE
Algolia: Please add auto complete feature

### ISSUE THREAD

**Issue by abdullah432** [knoxpo/dart_algolia#65]
Please implement the auto-complete algolia feature. 
https://www.algolia.com/doc/ui-libraries/autocomplete/guides/using-instantsearch/

**Comment by nhathiwala** [comment]
Nice suggestion, sure we will work on this feature.

**Comment by submetu** [comment]
Yes we need this feature :( It is making us think of going for another service besides algolia :(

**Comment by polarby** [comment]
Any update?

**Comment by tidianeb5** [comment]
@nhathiwala hello , any update about this issues

**Comment by nhathiwala** [comment]
@tidianeb5  Hey, I will surely try removing some time and try working on some concepts related to it.
